apps/web/src/components/Charts/Transaction/DailyAvgMaxBlobGasFeeChart.tsx

```diff
@@ -25,13 +25,10 @@ export const DailyAvgMaxBlobGasFeeChart: FC<
     ...buildTimeSeriesOptions({
       dates: days,
       axisFormatters: {
-        yAxisTooltip: (value) => formatWei(value, findBestUnit(value)),
+        yAxisTooltip: (value) => formatWei(value, unit, { displayUnit: true }),
```

Comment:
Don't we use the best unit anymore?

Reply:
We're still using the best unit. We're just taking it from this hook result a few lines [above](https://github.com/Blobscan/blobscan/blob/2589762d225070b63e3f3639378127a60e243f26/apps/web/src/components/Charts/Transaction/DailyAvgMaxBlobGasFeeChart.tsx#L20) to keep consistency with the label value